gradle.properties

```diff
@@ -1,5 +1,15 @@
 # Set heap size
```

Comment:
nit: Can we update this comment to explain why the --add-exports are there too please? Doesn't have to be in huge detail but if we know why it was added we can better evaluate if we can remove them in the future.

Reply:
Done.